code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -716,7 +803,9 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/gun/launcher/grenade/cyborg(src)
 	src.modules += new /obj/item/weapon/crowbar(src)
 	src.modules += new /obj/item/weapon/card/emag(src)
+	src.modules += new /obj/item/weapon/grenade/empgrenade(src) // an unusual thing, but.. yeah.. may work in cooperation

```

Comment:
Sounds like a bad idea from the view of the borg itself. Too high a chance to nuke oneself?